public/ifc2sql.py

```
@@ -0,0 +1,707 @@
+from typing_extensions import assert_never
+
+SQLTypes = typing.Literal["SQLite", "MySQL"]
+
+if TYPE_CHECKING:
+    import sqlite3
+    import mysql.connector
+    import mysql.connector.abstracts
+else:
+    try:
+        import sqlite3
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)